staging/kubefed/charts/controllermanager/templates/install-crds.yaml

```diff
@@ -0,0 +1,84 @@
+{{ if (or (or (not .Values.global.scope) (eq .Values.global.scope "Cluster")) (not (.Capabilities.APIVersions.Has "core.kubefed.io/v1beta1"))) }}
```

Comment:
Is this the right logic? 
Even if the cluster has this capability, we still want it to run. This will allow the upgrading of installed CRDs.

Reply:
good catch. i changed one of the conditions, but forgot the other....

Reply:
changed.